src/azure-openapi-validator/tests/composite-azure-test.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes } from '../rule';
+import { Message } from '../../jsonrpc/types';
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
+import { AutoRestPluginHost } from "../../jsonrpc/plugin-host";
+import { run } from "../../azure-openapi-validator";
+import {
+  assertValidationRuleCount,
+  collectTestMessagesFromValidator
+} from './utilities/tests-helper';
+import { DescriptionMustNotBeNodeName } from '../rules/DescriptionMustNotBeNodeName';
+
+@suite class CompositeAzureTests {
+  @test @timeout(120000) async "description should not be parameter name"() {
```

Comment:
note that you can run mocha with a default timeout - consider changing, say, `gulp test` to use a default of 2min instead of restating it in every test :) I'd expect `@timeout` decorators to be the exception